lib/linters/property_ordering.js

```diff
@@ -8,12 +8,20 @@ module.exports = {
     message: 'Property ordering is not alphabetized',
 
     lint: function propertyOrderingLinter (config, node) {
-        // Only support alpha for now
-        if (config.style !== 'alpha') {
+        let ordering;
+
+        try {
+            ordering = require('../config/ordering/' + config.style);
+        } catch (excep) {
             throw new Error(`Invalid setting value for propertyOrdering: ${ config.style }`);
         }
 
+        if (config.style !== 'alpha') {
```

Comment:
I think we can remove this check and use `util.format` when reporting the error instead. Take a look at `stringQuotes` for example.

Reply:
Yep, seems better